README.md

```diff
@@ -30,11 +30,17 @@ and the steps recommended for any base box.
 Usage
 -----
 
+dragon788 has built the VMware and Virtualbox versions on Hashicorp's [Atlas build service](https://atlas.hashicorp.com)
+
 ### VirtualBox Provider
 
 Assuming that you already have Packer,
-[VirtualBox](https://www.virtualbox.org/), and Vagrant installed, you
-should be good to clone this repo and go:
+[VirtualBox](https://www.virtualbox.org/), and Vagrant installed, you can jump right in
+with:
+
+    vagrant init dragon788/arch-ala-elasticdog; vagrant up --provider virtualbox
```

Comment:
Should we publish an 'official' packer-arch vagrant files here, rather than use @dragon788's version?

This way, @dragon788 can do his thing without worrying about messing up the upstream project.